repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {},
```

Comment:
How does the history service find the Mesos master?  We probably need some config here.  Also, how about the port that the REST service binds to?